src/common/components/resource.h

```diff
@@ -50,6 +51,12 @@ struct Unit {
 
 struct Good {};
 
+//See SysPopulationConsumption for an explanation of these values
+struct ConsumerGood {
+    double autonomous_consumption;
+    double marginal_propensity;
```

Comment:
Even if you put on top of the class where to look for documentation, can you put a short explanation for what these variables do?